### PR TITLE
Create Spring Boot-based archetype (Latest)

### PR DESCRIPTION
**Pull request by tarekoraby**

### CODE REVIEW

File: vaadin-archetype-spring-application/src/main/resources/archetype-resources/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# My App
+
+This project can be used as a starting point to create your own Vaadin application with Spring Boot.
+It contains all the necessary configuration and some placeholder files to get you started.
+
```
Comment (manolo) [inline vaadin-archetype-spring-application/src/main/resources/archetype-resources/README.md:5]: Perhaps we can mention here, that for a full-configured project they can visit http://start.vaadin.com ?

File: vaadin-archetype-spring-application/pom.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>com.vaadin</groupId>
+        <artifactId>vaadin-archetypes</artifactId>
+        <version>20.0.3</version>
```
Comment (manolo) [inline vaadin-archetype-spring-application/pom.xml:10]: This should be aligned with `vaadin-archetype-application` probably both should have `20.0-SNAPSHOT`

**Review by manolo — CHANGES_REQUESTED** [review]

File: vaadin-archetype-spring-application/src/main/resources/archetype-resources/README.md (hunk 1 above)
Comment (tarekoraby) [reply vaadin-archetype-spring-application/src/main/resources/archetype-resources/README.md:5]: done

File: vaadin-archetype-spring-application/pom.xml (hunk 2 above)
Comment (tarekoraby) [reply vaadin-archetype-spring-application/pom.xml]: done

**Review by manolo — APPROVED** [review]

**Pull request merged**